### PR TITLE
NYU: re-added Cho (meets criteria), added Ranganath

### PR DESCRIPTION
**Pull request by deniszorin**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -13302,4 +13302,6 @@ Yubao Wu,Georgia State University,http://cs.gsu.edu/profile/yubao-wu/,Ussl738AAA
 Zhisheng Yan,Georgia State University,https://grid.cs.gsu.edu/~zyan/,i6ePuTMAAAAJ
 Alex Zelikovsky,Georgia State University,http://cs.gsu.edu/profile/alex-zelikovsky/,UzCRNJwAAAAJ
 Yanqing Zhang,Georgia State University,https://grid.cs.gsu.edu/zhang/,NOSCHOLARPAGE
-Ying Zhu,Georgia State University,https://sites.google.com/site/yingzhugsu/,sz_Z3igAAAAJ
\ No newline at end of file
+Ying Zhu,Georgia State University,https://sites.google.com/site/yingzhugsu/,sz_Z3igAAAAJ
++Kyunghyun Cho,New York University,http://www.kyunghyuncho.me/,0RAmmIAAAAAJ^M
```
Comment (emeryberger) [inline csrankings.csv]: Spurious "+" here and in the next entry.

Comment (emeryberger) [reply csrankings.csv]: This person is at Facebook. Half-time employed faculty are not eligible for inclusion.
http://www.kyunghyuncho.me/

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (deniszorin) [reply csrankings.csv]: He is consulting at facebook, 1 day a week.  His NYU position is full time

Comment (kyunghyuncho) [reply csrankings.csv]: to reiterate @deniszorin's comment, i am a **full**-time faculty at NYU supervising 13 PhD students, teaching two full courses a year, running various industry/federal projects.

Comment (emeryberger) [reply csrankings.csv]: Great, thanks for the clarification (the number of students, etc. is immaterial -- it's all about the full-time appointment). 1 day a week is fine. The spurious "+" signs still need to be fixed.

**Comment by emeryberger** [comment]
Bump @deniszorin

File: csrankings.csv (hunk 1 above)
Comment (deniszorin) [reply csrankings.csv]: removed spurious "+",  added new faculty

**Comment by emeryberger** [comment]
Please resolve conflicts (pull again from master or open a new PR) and I will merge. Thanks.

**Comment by deniszorin** [comment]
Conflicts resolved.

**Review by emeryberger — APPROVED** [review]

**Comment by emeryberger** [comment]
Thanks!

**Pull request merged**